internal/databases/redis/archive/fs.go

```diff
@@ -0,0 +1,27 @@
+package archive
+
+import (
+	"io/fs"
+	"os"
+)
+
+type Folder struct {
+	Path     string
+	fileMode fs.FileMode
+}
+
+func CreateFolder(path string, fileMode fs.FileMode) *Folder {
+	return &Folder{
+		Path:     path,
+		fileMode: fileMode,
+	}
```

Comment:
Name is misleading - it doesn't create folder =(

Reply:
thx, gonna change this in next patch